locales/en-US/app.ftl

```diff
@@ -728,6 +728,10 @@ TransformNavigator--collapse-function-subtree = Collapse subtree: { $item }
 #   $host (String) - The "host" part of the URL, e.g. hg.mozilla.org
 SourceView--loading-url = Waiting for { $host }…
 
+# Displayed while the source view is waiting for the browser to deliver
+# the source code.
+SourceView--loading-browser-connection = Waiting for { -firefox-brand-name }…
```

Comment:
This looks quite generic, what do you think of something more specific like `Waiting for the source code from { -firefox-brand-name }…` ?

Reply:
Do we need "from Firefox" at all at that point?

Reply:
The various `Waiting for ...` messages are all displayed in the source view. We're always waiting for source code. The interesting part of the message is who we're waiting for to get that source code. And that's either the browser (or, more specifically, a privileged browser script which is computing the response to our WebChannel message) or a network request.

Reply:
I'm going to leave this as-is. We can tweak it later.